### DISCUSSION TITLE
Monitoring - timestamp block found

### DISCUSSION THREAD

**Discussion by jvxis** [onemorebsmith/kaspa-stratum-bridge#50] [Q&A]
Hi @onemorebsmith, how to have a timestamp of the moment a block is found?
![image](https://user-images.githubusercontent.com/108929149/193357615-146685ec-98a3-4447-9f4a-4130cc5a383b.png)

**Comment by jvxis** [comment]
Adding: Any way to transform the bluecore number into a link to kaspa explorer?

**Reply by onemorebsmith** [reply]
It's on my todo list, I'll get it in a day or two

**Comment by jvxis** [comment]
Another idea: 
![image](https://user-images.githubusercontent.com/108929149/193413111-6a10e924-a612-47ef-bb4a-2824a3316dd6.png)
A block finder rank
